src/components/App/App.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { TodoList } from '../TodoList/TodoList';
+import './App.css';
+
+import Users from '../../api/users';
+import Todos from '../../api/todos';
+
+const greaterId = Todos.sort((todo1, todo2) => todo2.id - todo1.id)[0].id;
+
+export class App extends React.PureComponent {
+  state = {
+    todos: Todos,
```

Comment:
prepare a new array of todos with users and use it 
no need to pass both todos AND users